sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EditEventPage.java

```diff
@@ -0,0 +1,100 @@
+      By.cssSelector("#eventInvestigationStatus label");
+  public static final By EVENT_INVESTIGATION_STATUS_INPUT =
+      By.cssSelector("#eventInvestigationStatus input:checked");
+  public static final By DISEASE_COMBOBOX = By.cssSelector("#disease div");
+  public static final By DISEASE_INPUT = By.cssSelector("#disease input");
+  public static final By EXTERNAL_ID_INPUT = By.cssSelector(" #externalId");
+  public static final By INTERNAL_ID_INPUT = By.cssSelector(" #internalId");
+  public static final By EXTERNAL_TOKEN_INPUT = By.cssSelector(" #externalToken");
+  public static final By DESCRIPTION_INPUT = By.cssSelector(" #eventDesc");
+  public static final By SOURCE_TYPE_COMBOBOX = By.cssSelector(" #srcType div");
+  public static final By SOURCE_TYPE_INPUT = By.cssSelector(" #srcType input");
+  public static final By TYPE_OF_PLACE_COMBOBOX = By.cssSelector(" #typeOfPlace div");
+  public static final By TYPE_OF_PLACE_INPUT = By.cssSelector(" #typeOfPlace input");
+  public static final By COUNTRY_COMBOBOX = By.cssSelector(" #country div");
+  public static final By REGION_COMBOBOX = By.cssSelector(" #region div");
+  public static final By DISTRICT_COMBOBOX = By.cssSelector(" #district div");
+  public static final By COMMUNITY_COMBOBOX = By.cssSelector(" #community div");
+  public static final By STREET_INPUT = By.cssSelector(" #street");
+  public static final By HOUSE_NUMBER_INPUT = By.cssSelector(" #houseNumber");
+  public static final By ADDITIONAL_INFORMATION_INPUT = By.cssSelector("#additionalInformation");
+  public static final By POSTAL_CODE_INPUT = By.cssSelector("#postalCode");
+  public static final By AREA_TYPE_COMBOBOX = By.cssSelector("#areaType div");
+  public static final By COMMUNITY_CONTACT_PERSON_COMBOBOX = By.cssSelector("#details");
+  public static final By CITY_INPUT = By.cssSelector("#city");
+  public static final By GPS_LATITUDE_INPUT = By.cssSelector("#latitude");
+  public static final By GPS_LONGITUDE_INPUT = By.cssSelector(" #longitude");
+  public static final By PRIMARY_MODE_OF_TRANSMISSION = By.cssSelector(" #diseaseTransmissionMode");
+  public static final By NOSOCOMIAL = By.cssSelector(" #nosocomial");
+  public static final By EVENT_INVESTIGATION_START_DATE =
+      By.cssSelector(" #eventInvestigationStartDate");
+  public static final By EVENT_INVESTIGATION_END_DATE =
+      By.cssSelector(" #eventInvestigationEndDate");
+  public static final By SOURCE_MEDIA_WEBSITE = By.cssSelector(" #srcMediaWebsite");
+  public static final By SOURCE_MEDIA_NAME = By.cssSelector(" #srcMediaName");
+  public static final By SOURCE_MEDIA_DETAILS = By.cssSelector(" #srcMediaDetails");
+  public static final By SOURCE_FIRST_NAME = By.cssSelector(" #srcFirstName");
+  public static final By SOURCE_LAST_NAME = By.cssSelector("#srcLastName");
+  public static final By SOURCE_TEL_NO = By.cssSelector(" #srcTelNo");
+  public static final By SOURCE_EMAIL = By.cssSelector("#srcEmail");
```

Comment:
Please add to the final of the variable name: INPUT, COMBOBOX or OPTIONS depends on the selector.